### ISSUE TITLE
Content type not set correctly for WASM

### ISSUE THREAD

**Issue by timheuer** [Azure/Storage-Feedback#3]
**Describe the bug**
When using Azure Storage v2 mode with static site hosting, certain files are missing default/correct content type definitions.  When uploading a WebAssembly project, the .wasm files are tagged with content type ```application/x-octet-stream``` and they should be marked as ```application/wasm```

**To Reproduce**
PRE-REG: Have an Azure Storage account with static site hosting enabled
1. Create a WebAssembly application (Blazor WASM)
2. Publish app output to a folder
3. Upload published output to $web storage blob container

**Expected behavior**
When browsing to the site, it should work.

**Actual behavior**
It does not work throwing a console error on incorrect MIME type.

**Screenshots**
<img width="577" alt="test" src="https://user-images.githubusercontent.com/4821/70583658-43266480-1b73-11ea-942c-76ff3668217c.png">

**Temporary Workaround**
Using CLI and/or Azure Storage Explorer, set the content-type specifically on that file.

**Comment by cveld** [comment]
Same counts for the new Azure Static Web App service. I don't know if there is a way to set mime-types for content that lives inside this new type of resource. The impact is that at least the stack I am currently using being mono wasm packager has to load the webassembly a second time using a fallback method. Therefore slowing down the booting of the app.

**Comment by timheuer** [comment]
@cveld this isn't what I'm experiencing with Blazor and the content serves correctly.  Are you using a different framework than Blazor?  See: https://timheuer.com/blog/hosting-blazor-in-azure-static-web-apps/

**Comment by cveld** [comment]
@timheuer yes I am experimenting with the mono wasm sdk. Maybe the wasm loader of Blazor is more fault tolerant against a wrong mimetype? Can you verify that you get back application/wasm?
I see that App Service is returning the right mimetype.

**Comment by timheuer** [comment]
@cveld you are right...we are working on the Static Web App support for blazor so I'll make sure we note this.  It 'works' for me (https://victorious-stone-0284aec1e.azurestaticapps.net/) but I think this is indeed due to some smart fallbacks for us :-)

**Comment by cveld** [comment]
@timheuer have you seen the console log error?
```
blazor.webassembly.js:1 Uncaught (in promise) Error: Blazor has already started.
    at Object.<anonymous> (blazor.webassembly.js:1)
    at blazor.webassembly.js:1
    at Object.next (blazor.webassembly.js:1)
    at blazor.webassembly.js:1
    at new Promise (<anonymous>)
    at r (blazor.webassembly.js:1)
    at Object.v [as start] (blazor.webassembly.js:1)
    at (index):23
(anonymous) @ blazor.webassembly.js:1
(anonymous) @ blazor.webassembly.js:1
(anonymous) @ blazor.webassembly.js:1
(anonymous) @ blazor.webassembly.js:1
r @ blazor.webassembly.js:1
v @ blazor.webassembly.js:1
(anonymous) @ (index):23
blazor.webassembly.js:1 Streaming compilation failed. Falling back to ArrayBuffer instantiation.  TypeError: Failed to execute 'compile' on 'WebAssembly': Incorrect response MIME type. Expected 'application/wasm'.
```

Apparently the bootstrap setup of Blazor is reusing the already received dotnet.wasm.

If you take a look at my mono wasm sdk based example, you will see the dotnet.wasm file is fetched a second time: https://red-tree-0e13b5c03.azurestaticapps.net/
